### DISCUSSION TITLE
[Done] Please try to add Axmol to the runtime list of FairyGUI:

### DISCUSSION THREAD

**Discussion by aismann** [axmolengine/axmol#1359] [General]
@halx99 
Please try to add Axmol to the runtime list of FairyGUI:
![image](https://github.com/axmolengine/axmol/assets/8652787/0dd78630-d570-48b3-b751-ecda6c6d413b)
![image](https://github.com/axmolengine/axmol/assets/8652787/ecf74d49-1f9d-4a9a-9c7d-d7934ad84928)

**Comment by halx99** [comment]
you need contact the author of fairygui

**Comment by aismann** [comment]
I have written an email.
I will closed it if it is done on FairyGUI page
https://fairygui.com/download#download-sdk

**Reply by halx99** [reply]
I convert to discussion since it not a issue

**Reply by aismann** [reply]
@halx99 plz remove the "help wanted"

**Comment by aismann** [comment]
I close it:
![image](https://github.com/axmolengine/axmol/assets/8652787/a75c9b67-a0b6-40ea-a31d-7ca09dbd6dac)
